publish/publish_workflow.wdl

```diff
@@ -247,171 +406,252 @@ task releaseHomebrew {
+        echo 'Create branch for release'
 
         git checkout -b ~{branchName} master
 
-        ########################################################
-        ##### Update url and hash for cromwell and womtool #####
-        ########################################################
-
-        # Find the line with the cromwell url. Also grab the next one that we assume will be the sha. Push both in an array
-        IFS=$'\r\n' command eval 'CROMWELL_ARRAY=($(grep -oh -A 1 'https://github.com/broadinstitute/cromwell/releases/download/.*/cromwell-.*\.jar' Formula/cromwell.rb))'
-        # Put the url and sha into separate variables. Also trim any leading space
-        CROMWELL_PREVIOUS_URL=$(echo "${CROMWELL_ARRAY[0]}" | sed -e 's/^[[:space:]]*//')
-        CROMWELL_PREVIOUS_SHA=$(echo "${CROMWELL_ARRAY[1]}" | sed -e 's/^[[:space:]]*//')
-
-        # Same for womtool
-        IFS=$'\r\n' command eval 'WOMTOOL_ARRAY=($(grep -oh -A 1 'https://github.com/broadinstitute/cromwell/releases/download/.*/womtool-.*\.jar' Formula/cromwell.rb))'
-        WOMTOOL_PREVIOUS_URL=$(echo "${WOMTOOL_ARRAY[0]}" | sed -e 's/^[[:space:]]*//')
-        WOMTOOL_PREVIOUS_SHA=$(echo "${WOMTOOL_ARRAY[1]}" | sed -e 's/^[[:space:]]*//')
-
-        # Compute the SHA of the newly released jars
-        CROMWELL_RELEASE_SHA=$(shasum -a 256 ~{cromwellJar} | cut -c 1-64)
-        WOMTOOL_RELEASE_SHA=$(shasum -a 256 ~{womtoolJar} | cut -c 1-64)
-
-        # Update cromwell url
-        sed -i '' -e "s;${CROMWELL_PREVIOUS_URL};~{cromwellReleaseUrl};g" Formula/cromwell.rb
-        # Update womtool url
-        sed -i '' -e "s;${WOMTOOL_PREVIOUS_URL};~{womtoolReleaseUrl};g" Formula/cromwell.rb
-        # Update cromwell sha
-        sed -i '' -e "s;${CROMWELL_PREVIOUS_SHA};sha256 \"${CROMWELL_RELEASE_SHA}\";" Formula/cromwell.rb
-        # Update womtool sha
-        sed -i '' -e "s;${WOMTOOL_PREVIOUS_SHA};sha256 \"${WOMTOOL_RELEASE_SHA}\";" Formula/cromwell.rb
-
-        ########################################################
-        #####   Verify install works and create PR if so   #####
-        ########################################################
+        echo '########################################################'
+        echo '##### Update url and hash for cromwell and womtool #####'
+        echo '########################################################'
+
+        echo 'Find the line with the cromwell url. Also grab the next one that we assume will be the sha.'
+        echo 'Push both in an array'
+        cromwell_array=()
+        while IFS='' read -r line; do cromwell_array+=("${line}"); done \
+        < <( \
+            grep \
+                -h -A 1 \
+                'https://github.com/broadinstitute/cromwell/releases/download/.*/cromwell-.*\.jar' \
+                Formula/cromwell.rb \
+            | awk -F '"' '{print $2}' \
+        )
+        # Put the url and sha into separate variables.
+        cromwell_previous_url="${cromwell_array[0]}"
```

Comment:
Any reason for lowercasing these? IMO they look more less like bash variables and more like WDL values this way

Reply:
`export`ed shell variables should be uppercase. All other local shell variables should be lowercase.

Ex: https://unix.stackexchange.com/questions/42847/are-there-naming-conventions-for-variables-in-shell-scripts

To drive you even further insane, I couldn't stand the mixture of WDL variable cases. So I moved *all* the WDL variables away from 🐍 towards 🐫.